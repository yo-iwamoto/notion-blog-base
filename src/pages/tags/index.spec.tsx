import { render } from '@/__tests__/util';
import Page from './index.page';

describe('[page] /tags', () => {
  it('renders /tags unchanged', async () => {
    const { container } = render(<Page tags={['React', 'Testing', 'Library']} />);

    expect(container).toMatchSnapshot();
  });
});
