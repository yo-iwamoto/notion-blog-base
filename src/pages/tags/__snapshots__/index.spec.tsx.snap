// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`[page] /tags renders /tags unchanged 1`] = `
<div>
  <div
    class="font-sans text-gray-900"
  >
    <div
      class="mx-4 mt-12"
    >
      <div
        class="mx-auto w-full max-w-2xl"
      >
        <main>
          <h1
            class="mb-12 text-4xl font-bold"
          >
            Tags
          </h1>
          <ul>
            <li
              class="my-4"
            >
              <span
                class="cursor-pointer text-2xl font-bold italic before:content-[\\"#\\"] hover:text-gray-600"
              >
                React
              </span>
            </li>
            <li
              class="my-4"
            >
              <span
                class="cursor-pointer text-2xl font-bold italic before:content-[\\"#\\"] hover:text-gray-600"
              >
                Testing
              </span>
            </li>
            <li
              class="my-4"
            >
              <span
                class="cursor-pointer text-2xl font-bold italic before:content-[\\"#\\"] hover:text-gray-600"
              >
                Library
              </span>
            </li>
          </ul>
        </main>
      </div>
    </div>
  </div>
</div>
`;
